alongtheway.txt

```diff
@@ -678,7 +678,7 @@ Day 3: transfiguration
 
 Read: Matthew 17:1-8
 
-Reflect: Jesus invited Peter, James and John up to the mountain where he gave them a sneak peek of his glory. I love Peter's response in this moment because it is so honest. He asked to live in this moment. And that can be my response too when I'm feeling close to God -- I just want to pitch my tent right here because it's so amazing. But Jesus took them back down the mountain and back down into ministry. I believe mountain-top moments are not where we live. They are special moments to prepare us for our calling -- moments when we reorient ourselves and get vision of what God's heart is for us right now. Then we get to walk back down the mountain and walk it out.
+Reflect: Jesus invited Peter, James and John up to the mountain where he gave them a sneak peek of his glory. I love Peter's response here because it is so honest. He asked if he could live in that moment. And that can be my response too when I'm feeling close to God -- I just want to pitch my tent right here because it's so amazing. But Jesus took them back down the mountain and back down into ministry. I believe mountain-top moments are not where we live. They are special moments to prepare us for our calling -- moments when we reorient ourselves and get vision of what God's heart is for us right now. Then we get to walk back down the mountain and walk it out.
```

Comment:
from
"I love Peter's response in this moment because it is so honest. He asked to live in this moment." 
to
"I love Peter's response here because it is so honest. He asked if he could live in that moment."